lsp-fsharp.el

```diff
@@ -0,0 +1,80 @@
+;;; lsp-fsharp.el --- description -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2019  Reed Mullanix
+
+;; Author: Reed Mullanix <reedmullanix@gmail.com>
+;; Keywords:
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; lsp-fsharp client
+
+;;; Code:
+
+(require 'lsp-mode)
+(require 'pcase)
+
+(defgroup lsp-fsharp nil
+  "LSP support for the F# Programming Language, using the FsharpAutoComplete server."
+  :link '(url-link "https://github.com/fsharp/FsAutoComplete")
+  :group 'lsp-mode)
+
+(defcustom lsp-fsharp-server-runtime 'net-core
+  "The .NET runtime to use."
+  :group 'lsp-fsharp
+  :type '(choice (const :tag "Use .Net Core" 'net-core)
+                 (const :tag "Use Mono" 'mono)
+                 (const :tag "Use .Net Framework" 'net-framework)))
+
+(defcustom lsp-fsharp-server-path nil
+  "The path to fsautocomplete."
+  :group 'lsp-fsharp
+  :risky t
+  :type 'file)
+
+
+(defcustom lsp-fsharp-server-args nil
+  "Extra arguments for the F# language server."
+  :type '(repeat string)
+  :group 'lsp-fsharp)
+
+
+(defun lsp-fsharp--fsac-runtime-cmd ()
+  "Get the command required to run fsautocomplete based off of the current runtime."
+  (pcase lsp-fsharp-server-runtime
+    ('net-core '("dotnet"))
+    ('mono '("mono"))
+    ('net-framework nil)))
+
+(defun lsp-fsharp--make-launch-cmd ()
+  "Build the command required to launch fsautocomplete."
+  (if lsp-fsharp-server-install-dir (lsp-warn "Cannot locate fsautocomplete"))
```

Comment:
```suggestion
  (if lsp-fsharp-server-path (lsp-warn "Cannot locate fsautocomplete"))
```

should this be `lsp-fsharp-server-path` instead of `lsp-fsharp-server-install-dir`, since you renamed it a few commits ago?

Reply:
🤦‍♂ Nice catch, fixed.